README.md

```diff
@@ -154,6 +154,9 @@ javabind recognizes several widely-used types and marshals them automatically be
 | `int16_t` | `short` | `short` |
 | `int32_t` | `int` | `int` |
 | `int64_t` | `long` | `long` |
+| `uint8_t` | `short` | `short` |
+| `uint16_t` | `int` | `int` |
+| `uint32_t` | `long` | `long` |
```

Comment:
I have mixed opinion about automatic widening conversions. In our specific data engineering use cases, we want to ensure that the C++ integer type fits exactly in the Java integer width, and we want to avoid widening conversions. However, I can imagine use cases where this is the preferred approach. I can also imagine use cases where we want to retain the integer width and are less sensitive to an implicit signed/unsigned conversion. (This is how `javabind` works for pointers.) A compromise solution may be to hide this feature behind a preprocessor flag, e.g. `JAVABIND_INTEGER_WIDENING_CONVERSION`.

Reply:
When the option JAVABIND_INTEGER_WIDENING_CONVERSION isn't set, support for unsigneds would be disabled then as it is currently?

Reply:
Yeah, that's what I would go for. In our projects, passing unsigned integers from C++ to Java constitutes an error because it could lead to data mutation (large value flips to negative) or trigger an implicit widening conversion (extra space when significant amount of data is involved).

By defining `JAVABIND_INTEGER_WIDENING_CONVERSION`, a library user could choose to consciously opt in to this behavior.

Reply:
Added option JAVABIND_INTEGER_WIDENING_CONVERSION, default value is OFF.